Documentation/configuration.md

```diff
@@ -152,44 +140,35 @@ configuration and the following local configurations:
-encoded in base64). For `tectonic.com`, it will send 
-`Authorization: Bearer tectonic-token`.
+The result is that when downloading data from `kubernetes.io`, `rkt` still sends `Authorization: Bearer common-token`, but when downloading from `coreos.com`, it sends `Authorization: Basic Zm9vOmJhcg==` (i.e. `foo:bar` encoded in base64).
+For `tectonic.com`, it will send `Authorization: Bearer tectonic-token`.
 
-Note that _within_ a particular configuration directory (either system or
-local), it is a syntax error for the same domain to be defined in multiple
-files.
+Note that _within_ a particular configuration directory (either system or local), it is a syntax error for the same domain to be defined in multiple files.
 
 ### rktKind: `dockerAuth`
 
-This kind of configuration is used to set up necessary credentials when
-downloading data from Docker registries. The configuration files should be
-placed inside `auth.d` subdirectory (e.g. in `/usr/lib/rkt/auth.d` or
-`/etc/rkt/auth.d`).
+This kind of configuration is used to set up necessary credentials when downloading data from Docker registries.
+The configuration files should be placed inside `auth.d` subdirectory (e.g. in `/usr/lib/rkt/auth.d` or `/etc/rkt/auth.d`).
 
 #### rktVersion: `v1`
 
 ##### Description and examples
 
-This version of `dockerAuth` configuration specifies two additional fields:
-`registries` and `credentials`.
+This version of `dockerAuth` configuration specifies two additional fields: `registries` and `credentials`.
 
-The `registries` field is an array of strings describing Docker registries for
-which the associated credentials should be used. A short list of popular Docker
-registries is below. This field must be specified and cannot be empty.
+The `registries` field is an array of strings describing Docker registries for which the associated credentials should be used.
+A short list of popular Docker registries is below.
+This field must be specified and cannot be empty.
 
-`credentials` field holds the necessary data to authenticate against the
-Docker registry. This field must be specified and cannot be empty.
+`credentials` field holds the necessary data to authenticate against the Docker registry.
+This field must be specified and cannot be empty.
 
-Currently, Docker registries only support basic HTTP authentication, so
-`credentials` field has two subfields - `user` and `password`. These
-fields must be specified and cannot be empty.
+Currently, Docker registries only support basic HTTP authentication, so `credentials` field has two subfields - `user` and `password`.
+These fields must be specified and cannot be empty.
 
 Some popular Docker registries:
-* index.docker.io (this is used when no Docker registry is
-  specified in a reference on the rkt command line, as in `docker://redis`)
+
+* index.docker.io (this is used when no Docker registry is specified in a reference on the rkt command line, as in `docker://redis`)
```

Comment:
* index.docker.io (Assumed as the default when no specific registry is named on the rkt command line, as in `docker:///redis`.)